CRVResponse/src/CrvCoincidenceFinder_module.cc

```diff
@@ -757,7 +758,7 @@ namespace mu2e
                       [](L &a, L &b){return a->_maxSlope < b->_maxSlope;}))->_maxSlope;
     double maxSlopeDifference = (*std::max_element(layerIterators,layerIterators+n,
                                 [](L &a, L &b){return a->_maxSlopeDifference < b->_maxSlopeDifference;}))->_maxSlopeDifference;
-    double slopes[n-1];
+    std::array<double,3> slopes;
```

Comment:
Let's wrap this up.  I am uncomfortable leaving it as it since the 3 is a magic number and, in principle, could need maintenance in the future.  So let's put this one back to vector.  Sorry for the noise.  Ray, can you look after this.

Reply:
We already have `constexpr static const int nLayers = 4;` in line 163.
So we could use nLayers to create `constexpr int maxSlopes = (nLayers-1);`

Reply:
DataProducts/inc/CRVId.hh is the logical place for this, but it does not have nlayers. Let me know if you want it there, or Ralf's suggestion.

Reply:
I like Rays idea of move the constexpr to CRVid.hh, removing it from this file and then using std::array with maxSlopes.

@ehrlich-uva is that OK with you?  

In a separate PR we can scrub for other definitions of nLayers  .

Reply:
Yes, that's Ok with me.